angular.json

```diff
@@ -141,8 +141,5 @@
       }
     }
   },
-  "defaultProject": "Angularflix-portal",
```

Comment:
It doesn't make difference if you commit erasing the key, the key is still here. You'll have to make a cherry pick and undo your commit to remove it completely from the history.